tests/testthat/test-data-table.R

```diff
@@ -5,11 +5,34 @@ test_that("use_data_table() requires a package", {
 
 test_that("use_data_table() Imports data.table", {
   create_local_package()
+  use_package_doc()
   with_mock(
     check_installed = function(pkg) TRUE,
+    roxygen_update_ns = function(...) NULL,
+    check_functions_exist = function(...) TRUE,
     use_data_table()
   )
   expect_match(desc::desc_get("Imports"), "data.table")
-  datatable_doc <- read_utf8(proj_path("R", "utils-data-table.R"))
-  expect_match(datatable_doc, "#' @import data.table", all = FALSE)
+  package_doc <- read_utf8(proj_path(package_doc_path()))
```

Comment:
Is this serving a purpose at the moment?

Reply:
You're right, this is ⚰️ code RIP 🙏 🌹